SharpCompress/Writer/Zip/ZipCompressionInfo.cs

```diff
@@ -0,0 +1,47 @@
+﻿using SharpCompress.Common;
+using SharpCompress.Common.Zip;
+using SharpCompress.Compressor.Deflate;
+
+namespace SharpCompress.Writer.Zip
+{
+    public class ZipCompressionInfo
```

Comment:
I think it would be best if we hide this class, because it is only necessary for the implementation of the `ZipWriter` class. It would be better to pass `CompressionInfo` in all public methods and keep `ZipCompressionInfo` for the private implementation. We can make the class `internal` or nest it inside the `ZipWriter` class as `private` for better encapsulation.